rust/template/src/lib.rs

```diff
@@ -61,6 +61,10 @@ use serde::Deserializer;
 use serde::Serialize;
 use serde::Serializer;
 
+// This import is only needed to convince the OS X compiler to export
+// `extern C` functions declared in ddlog_log.rs in the generated lib.
+pub use ddlog_log as hidden_ddlog_log;
```

Comment:
Maybe feature gate this under the `c_api` flag so it's not exported for non-ffi users

Reply:
good point